tests/setupMocks.ts

```diff
@@ -1,7 +1,7 @@
-import MockGrpcClient from './mockGrpcClient'
+import MockChronikClient from './mockBlockchainClient'
 
-jest.mock('grpc-bchrpc-node', () => ({
-  ...jest.requireActual('grpc-bchrpc-node'),
+jest.mock('chronik-client', () => ({
```

Comment:
Indeed, this removal was part of the beginning of this PR where I thought grpc would be removed, then I didn't think again about it since tests were to be addressed in the last PR of its own, but I addressed this by creating a new file for each client to be tested (just skeleton)